pex/pex_builder.py

```diff
@@ -455,11 +509,13 @@ def freeze(self, bytecode_compile=True):
     self._prepare_manifest()
     self._prepare_bootstrap()
     self._prepare_main()
+    if verify_entry_point:
+      self._verify_entry_point()
     if bytecode_compile:
       self._precompile_source()
     self._frozen = True
 
-  def build(self, filename, bytecode_compile=True):
+  def build(self, filename, bytecode_compile=True, verify_entry_point=False):
```

Comment:
This is the interface pants will call, correct?

changing it in a way to keep backwards compatibility.

Reply:
Yes, but see above.